src/sweeper/sweepers/duplicates.py

```diff
@@ -48,7 +48,7 @@ def sweep(self):
                     object_id = row[oid_index]
 
                     if shape_wkt is None:
```

Comment:
should the empty shape in the duplicate sweeper be a warning?

Reply:
I think I agree with your opinion in the second part of your comment.  It's probably nice to have duplicates and empties separate in case you want to focus on one or the other.  That way the empties sweeper can be used to specifically handle those issues, but the duplicates can ID them to make the user aware.

Reply:
I think we should keep them separated.  ie: check for dups in the duplicates.py and check for empties in the empties.py.  I'm not sure we even need it to make suggestions.  that's my two cents.  what do you think @eneemann @ZachBeck  ?

Reply:
I agree

Reply:
I'm fine with that

Reply:
okay.  do you want me to remove lines 50-53 from duplicates.py or should I put an issue in github to have that aspect removed (in case it's not just lines 50-53) ?
```
                    if shape_wkt is None:
                        # self.report['issues'].append(object_id)

                        continue
```

Reply:
nuke it

Reply:
done

Reply:
will you push your commits when you do this?

Reply:
commits are free. It's nice to see that you understood the change and commit it with your 'done' comment